articles/api/management/v1/use-cases.md

```diff
@@ -8,40 +8,17 @@ contentType: reference
 useCase: invoke-api
 ---
 
-# Management API v1 Use Cases
+# Features not affected by the Management API v1 Deprecation
```

Comment:
Yes. I just started a discussion about this. Originally, we intended to leave these endpoints outside of the deprecation (and therefore available as they were before) since we have no replacement for them. 

However, they only are used by ~16 tenants in cloud, so we might remove this doc completely and mark all for deprecation, and make an exception for those 16 tenants explicitly until we can provide an alternative.

Reply:
I'm all for removing them.  If we don't want people to use them, we should remove the public facing docs on them :)

Reply:
Sounds good to me. Though, we'll have 14 public cloud customers still using them for the immediate future and no idea how many PSaaS customers are using these at the moment. As long as it's a capability for some of our customers we need it documented somewhere. Could we extract this to the either the TF knowledge base or DSE's troubleshooting pages? As long as folks can easily find this info I'm happy to remove it from the public docs.

cc @mauricioschneider

Reply:
The docs for the endpoints will still be available here: https://auth0.com/docs/api/management/v1 (as they were), but we'll remove the 'use cases' docs and the endpoints once we reach EOL.